models/tsodyks_synapse.h

```diff
@@ -59,29 +60,30 @@ to reproduce the results of [1]_ and to use this model of synaptic plasticity
 in its original sense, the user therefore has to ensure the following
 conditions:
 
-1.) The postsynaptic neuron must be of type iaf_psc_exp or iaf_psc_exp_htum,
+1.) The postsynaptic neuron must be of type ``iaf_psc_exp`` or ``iaf_psc_exp_htum``,
 because these neuron models have a postsynaptic current which decays
 exponentially.
 
-2.) The time constant of each tsodyks_synapse targeting a particular neuron
+2.) The time constant of each ``tsodyks_synapse`` targeting a particular neuron
 must be chosen equal to that neuron's synaptic time constant. In particular
 that means that all synapses targeting a particular neuron have the same
-parameter tau_psc.
+parameter ``tau_psc``.
 
 However, there are no technical restrictions using this model of synaptic
 plasticity also in conjunction with neuron models that have a different
 dynamics for their synaptic current or conductance. The effective synaptic
 weight, which will be transmitted to the postsynaptic neuron upon occurrence
-of a spike at time t is u(t)*x(t)*w, where u(t) and x(t) are defined in
+of a spike at time t is :math:`u(t)*x(t)*w`, where u(t) and x(t) are defined in
 Eq (3) and (4), w is the synaptic weight specified upon connection.
-The interpretation is as follows: The quantity u(t)*x(t) is the release
+The interpretation is as follows: The quantity `u(t)*x(t)` is the release
 probability times the amount of releasable synaptic vesicles at time t of the
 presynaptic neuron's spike, so this equals the amount of transmitter expelled
 into the synaptic cleft.
+
 The amount of transmitter than relaxes back to 0 with time constant tau_psc
 of the synapse's variable y. Since the dynamics of y(t) is linear, the
 postsynaptic neuron can reconstruct from the amplitude of the synaptic
-impulse u(t)*x(t)*w the full shape of y(t). The postsynaptic neuron, however,
+impulse `u(t)*x(t)*w` the full shape of y(t). The postsynaptic neuron, however,
 might choose to have a synaptic current that is not necessarily identical to
 the concentration of transmitter y(t) in the synaptic cleft. It may realize
 an arbitrary postsynaptic effect depending on y(t).
```

Comment:
```suggestion
the concentration of transmitter `y(t)` in the synaptic cleft. It may realize
an arbitrary postsynaptic effect depending on `y(t)`.
```